sdk/search/Azure.Search.Documents/tests/Samples/Sample05_IndexingDocuments.cs

```diff
@@ -167,6 +167,8 @@ public async Task BufferedSender()
 
                 // Simple
                 {
+                    searchClient = GetOriginal(searchClient);
```

Comment:
this will keep the un-uninstrumented instance in searchClient variable. It that correct?

Reply:
Yes. To be used in the ctor below. Same work as done in the other test, except here we want to keep the call to `GetOriginal()` outside of the snippet.